### DISCUSSION TITLE
Error in optimization of NACA0012 case with DAFoam 3.0.2

### DISCUSSION THREAD

**Discussion by uttamcadambi07** [mdolab/dafoam#370] [Tutorials]
I installed DAFoam 3.0.2 on Ubuntu 20.04 LTS using a virtual machine. 

While instalkling DAFoam, I encountered with aero w.r.t. MPhysAerostruct (find link to the thread here: https://github.com/mdolab/dafoam/discussions/344#discussioncomment-3865411)

I ignored it since I am not doing aerostructural optimization.

However, when I tried to run the test case from the 2022 workshop on NACA0012, I got the following  

[naca0012_error.txt](https://github.com/mdolab/dafoam/files/9776784/naca0012_error.txt)

What seems to be the issue? My OpenMDAO works fine (I ran the runScript.py from mdao directory). Has this error got something to do with the failed aerostruct installation?

**Comment by uttamcadambi07** [comment]
It seems to me that there is some issue with wingFFD.xyz. Because it says that the ffd_file was not declared. Is there a line that should be added before OM_DVGEOCOMP(ffd_file="FFD/wingFFD.xyz")?

**Reply by friedenhe** [reply]
You need to use a specific version of DAFoam for the workshop case. Check the first workshop slide and use the corresponding DAFoam image.
